src/ContainerAppHelper.ts

```diff
@@ -270,6 +270,38 @@ export class ContainerAppHelper {
         }
     }
 
+    /**
+     * Gets the location of an existing Container App Environment
+     * @param environmentName - the name of the Container App Environment
+     * @param resourceGroup - the resource group that the Container App Environment is found in
+    */
+    public async getExistingContainerAppEnvironmentLocation(environmentName: string, resourceGroup: string) {
+        try {
+            let command = `az containerapp env show -g ${resourceGroup} --query location -n ${environmentName}`;
+            let executionResult = await util.execute(command);
+            return executionResult.exitCode === 0 ? executionResult.stdout.toLowerCase().replace(/["() ]/g, "").trim() : null;
+        } catch (err) {
+            toolHelper.writeInfo(err.message);
+            return null;
+        }
+    }
+
+    /**
+     * Gets the environment Id of an existing Container App
```

Comment:
nit: _Gets the environment **name** of an existing Container App_

Reply:
Updated.